src/main/java/wis/task/TaskList.java

```diff
@@ -0,0 +1,55 @@
+package wis.task;
+
+import java.io.BufferedWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+
+import wis.Storage;
+import wis.util.Printer;
+
+public class TaskList {
+    private ArrayList<Task> list;
```

Comment:
Consider using `tasks` as the variable name instead to describe what this list contains